src/common-messagelog/src/main/java/ee/ria/xroad/common/messagelog/archive/EncryptionConfig.java

```diff
@@ -40,6 +41,7 @@ public class EncryptionConfig {
     private final boolean enabled;
     private final Path gpgHomeDir;
     private final Set<String> encryptionKeys;
+    private final List<EncryptionMember> encryptionMembers;
```

Comment:
Consider making more null-safe. Perhaps make the list optional? Same could be considered for encryptionKeys